packages/core/src/middleware/koa-auth.ts

```diff
@@ -23,6 +33,22 @@ export default function koaAuth<StateT, ContextT>(): Middleware<
         { supportedTypes: [bearerToken] }
       )
     );
+    const jwt = authorization.slice(bearerToken.length + 1);
+
+    try {
+      const {
+        payload: { sub },
+      } = await jwtVerify(jwt, publicKey, {
+        issuer,
+        audience: adminResource,
+      });
+      assert(sub);
+      const user = await findUserById(sub);
+      ctx.user = pick(user, ...userInfoSelectFields);
+    } catch {
+      throw new RequestError({ code: 'auth.unauthorized', status: 401 });
```

Comment:
should we use constants instead of string variables on the `code` filed ?

Reply:
code is typed, not just string. see `packages/schemas/src/api/error.ts`